include/dd/Package.hpp

```diff
@@ -930,13 +966,15 @@ namespace dd {
             if (x.p == nullptr) return y;
             if (y.p == nullptr) return x;
 
-            if (x.w == Complex::zero) {
-                if (y.w == Complex::zero) return y;
+            if (x.w.approximatelyZero()) {
+                if (y.w.approximatelyZero()) {
+                    return Edge<Node>::zero;
```

Comment:
this is a significant change in the behavior of the `add2` function. Any particular reason for loosening the equality check here?

Reply:
I added a function `exactlyZero` at `complex.hpp` and the `complexTable.hpp` to compare exactly against the value zero within the table.